examples/video-chat/hooks/useForegroundService.ts

```diff
@@ -43,17 +38,10 @@ async function startForegroundService() {
   }
 }
 
-export function useJoinRoom({ isMicrophoneAvailable }: Props) {
-  const { startMicrophone } = useMicrophone();
-
-  const joinRoom = useCallback(async () => {
-    await startForegroundService();
-
-    await startMicrophone({
-      audioTrackMetadata: { active: isMicrophoneAvailable, type: 'audio' },
-      microphoneEnabled: isMicrophoneAvailable,
-    });
-  }, [isMicrophoneAvailable, startMicrophone]);
+export function useForegroundService() {
```

Comment:
There will be soon new PR that will update this hook, to contain all logic related to notifee